src/Posts/RepostButton.jsx

```diff
@@ -137,13 +134,13 @@ const promoteToBlog = () => {
 };
 
 const buildMenu = (accountId, blockHeight) => {
```

Comment:
@jackson-harris-iii I know there some dynamic logic to build the menu based on the post type, if it's straightforward, please reorder the menu options such that 'Activity Feed' is first and 'My Blog' is below.